contracts/libraries/VTokenPositionSet.sol

```diff
@@ -384,16 +391,16 @@ library VTokenPositionSet {
         VTokenPosition.Info storage position = set.positions[poolId];
 
         uint256 price = protocol.getVirtualTwapPriceX128(poolId);
```

Comment:
We should replace this with the checked price, right?

Reply:
I'm not exactly sure where we need to keep checked price and where we shouldn't. Can you please tell me where we should have (or where we should not, either works), else feel free to make this change yourself if you'd be making a PR in next two weeks.